From d08f1266bfd13d86fe53769e579c4a03f76c180b Mon Sep 17 00:00:00 2001
From: Konrad Weihmann <kweihmann@outlook.com>
Date: Sun, 26 Sep 2021 21:14:10 +0200
Subject: [PATCH] lift version requirements

remove version limitations for
- pluggy
- typing
- system provided libs for libarchive and co

Signed-off-by: Konrad Weihmann <kweihmann@outlook.com>
Upstream-Status: Inappropriate [configuration]
---
 requirements.txt | 5 +----
 setup.cfg        | 4 +---
 2 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index 301a02099..7d68a5208 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -15,8 +15,6 @@ cryptography==3.4.7
 debian-inspector==30.0.0
 dparse==0.5.1
 extractcode==21.7.23
-extractcode-7z==16.5.210531
-extractcode-libarchive==3.5.1.210531
 fasteners==0.16
 fingerprints==1.0.3
 ftfy==6.0.3
@@ -66,11 +64,10 @@ saneyaml==0.5.2
 six==1.16.0
 sortedcontainers==2.4.0
 soupsieve==2.2.1
-spdx-tools==0.7.0a3
+spdx-tools>=0.6.0
 text-unidecode==1.3
 toml==0.10.2
 typecode==21.6.1
-typecode-libmagic==5.39.210531
 typing==3.6.6
 typing-extensions==3.10.0.0
 urllib3==1.26.5
diff --git a/setup.cfg b/setup.cfg
index a7e9cc7a6..a885dc52b 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -84,7 +84,6 @@ install_requires =
     pdfminer.six >= 20200101
     pefile >= 2020.1.1
     pkginfo >= 1.7.0
-    pluggy >= 0.12.0, < 1.0
     plugincode >= 21.1.21
     publicsuffix2
     pyahocorasick >= 1.4.0, < 1.5
@@ -93,10 +92,9 @@ install_requires =
     pymaven_patch >= 0.2.8
     requests >= 2.7.0, < 3.0.0
     saneyaml >= 0.5.2
-    spdx_tools >= 0.7.0a3
+    spdx_tools >= 0.6.0
     text_unidecode >= 1.0, < 2.0
     toml >= 0.10.0
-    typing >=3.6, < 3.7; python_version < "3.7"
     urlpy
     xmltodict >= 0.11.0
     zipp >= 3.0.0; python_version < "3.9"
-- 
2.25.1

